packages-tests/BetterPhpDocParser/PhpDocParser/StaticDoctrineAnnotationParser/PlainValueParserTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\BetterPhpDocParser\PhpDocParser\StaticDoctrineAnnotationParser;
+
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\ParserFactory;
+use PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocTagNode;
+use PHPStan\PhpDocParser\Lexer\Lexer;
+use Rector\BetterPhpDocParser\PhpDocInfo\PhpDocInfoFactory;
+use Rector\BetterPhpDocParser\PhpDocInfo\TokenIteratorFactory;
+use Rector\BetterPhpDocParser\PhpDocParser\ClassAnnotationMatcher;
+use Rector\BetterPhpDocParser\PhpDocParser\StaticDoctrineAnnotationParser\PlainValueParser;
+use Rector\Core\PhpParser\Node\BetterNodeFinder;
+use Rector\Testing\PHPUnit\AbstractLazyTestCase;
+use Rector\Testing\TestingParser\TestingParser;
+
+class PlainValueParserTest extends AbstractLazyTestCase
```

Comment:
same with https://github.com/rectorphp/rector-src/pull/5123#pullrequestreview-1678697825 , I prefer placing fixture in `tests/Issues` so it is easier what config and rule that causing it.

You can create custom rule under tests/Issues, something like

https://github.com/rectorphp/rector-src/blob/a9908f6498004953531a1a145a4e271fa953a3c7/tests/Issues/PartialValueDocblockUpdate/Source/PartialUpdateTestRector.php#L17

then register in config test 

https://github.com/rectorphp/rector-src/blob/a9908f6498004953531a1a145a4e271fa953a3c7/tests/Issues/PartialValueDocblockUpdate/config/configured_rule.php#L9

along with other config if needed.